advanced/keyboard-shortcuts.rst

```diff
@@ -28,15 +23,158 @@ screenshot:
+====================================  ================================================
+Key / key combination                       Function
+====================================  ================================================
+``t``                                 Enable or disable the inline translation
+====================================  ================================================
+
+
+Appearance
+^^^^^^^^^^
+
+====================================  ================================================
+Key / key combination                       Function
+====================================  ================================================
+``d``                                 Switch between dark and light mode
+====================================  ================================================
+
+Tickets
+^^^^^^^
+
+====================================  ===================================================================
+Key / key combination                 Function
+====================================  ===================================================================
+``x``                                 Create a new note article
+``r``                                 Reply to the last article
+``i``                                 Switch the visibility of the article between internal and public
+``shift`` + ``c``                     Set state of the ticket to "closed"
+``←`` / ``→``                         Navigate through article
+``::``                                Insert text module (while composing an article)
+``??``                                Insert knowledge base article (while composing an article)
+``@@``                                Mention a user (while composing an article)
+====================================  ===================================================================
+
+Text Editing
+^^^^^^^^^^^^
+
+How
+   You can apply a text format *before* typing or *after* typing. Example:
+
+   Before typing:
+
+   * Press ``ctrl`` + ``i`` to enter *italics* mode,
+   * enter your desired text, and
+   * press ``ctrl`` + ``i`` again to return to normal text mode.
+
+   After typing:
+
+   * Enter your desired text,
+   * click-and-drag with the mouse to select it, and
+   * press ``ctrl`` + ``i`` to set the text in *italics*.
+
+Key Combinations
+   ==============================  =============================================
+   Key / key combination           Function
+   ==============================  =============================================
+   ``ctrl`` + ``u``                Format text underlined
+   ``ctrl`` + ``b``                Format text in **bold**
+   ``ctrl`` + ``i``                Format text in *italics*
+   ``ctrl`` + ``s``                Format text as  ̶s̶t̶r̶i̶k̶e̶t̶h̶r̶o̶u̶g̶h̶
+   ``ctrl`` + ``v``                Paste text from clipboard
+   ``ctrl`` + ``shift`` + ``v``    Paste text from clipboard (as plain text)
+   ``shift`` + ``ctrl`` + ``f``    Remove formatting of text
```

Comment:
Perhaps we should mention somewhere that the magic key (`ctrl`) is `cmd`, and the hotkey combination (`shift` + `ctrl`) is `ctrl` + `option` on Mac keyboards. Possibly in a note below this section or something.

Reply:
Is the combination `ctrl` + `option` or `cmd` + `option` for Mac?

Reply:
It's `ctrl` + `option`, just verified it. `cmd` is used only as the magic key (i.e. for editor text formatting).